pkg/cmd/attestation/verify/policy.go

```diff
@@ -68,9 +89,9 @@ func newEnforcementCriteria(opts *Options) (verification.EnforcementCriteria, er
 	// If the tenant option is provided, set the SourceRepositoryOwnerURI extension
 	// using the specific URI format
 	if opts.Tenant != "" {
-		c.Certificate.SourceRepositoryOwnerURI = fmt.Sprintf("https://%s.ghe.com/%s", opts.Tenant, opts.Owner)
+		c.Certificate.SourceRepositoryOwnerURI = fmt.Sprintf("https://%s.ghe.com/%s", opts.Tenant, owner)
```

Comment:
doesn't this duplicate `expandToGitHubURL`? Couldn't this just be a one liner with no `if else`?

Reply:
`expandToGitHubURL ` prefixes the URL with `(?i)^` for a regex but the rest of it is identical, I can update some functions to reduce the duplication.